ariel-gonzales/lesson-1/homework-1-contacts-manager/src/Contact.java

```diff
@@ -0,0 +1,8 @@
+
+/**
+ * contatcs_manager.Contact class.
+ */
+public class Contact {
+    String name;
+    String phoneNumber;
```

Comment:
These fields should be private and you should create its getter and setters methods